data/yam/agama/auto/template.libsonnet

```diff
@@ -41,5 +41,6 @@ function(addon_ha_reg_code='',
   [if storage == 'root_filesystem_ext4' then 'storage']: storage_lib['root_filesystem_ext4'],
   [if storage == 'root_filesystem_xfs' then 'storage']: storage_lib['root_filesystem_xfs'],
   [if storage == 'whole_disk_and_boot_unattended' then 'storage']: storage_lib['whole_disk_and_boot_unattended'],
+  [if storage == 'raid0' then 'storage']: storage_lib['raid0'],
```

Comment:
please, add the new lines in alphabetical order as much as possible.

Reply:
yes, updated, thanks.